apps/builder/app/builder/features/style-panel/style-source/use-sortable.tsx

```diff
@@ -81,10 +94,22 @@ export const useSortable = <Item extends { id: string }>({
         return false;
       }
 
-      return getItemId(closest) || false;
+      const itemId = getItemId(closest);
+      if (itemId === undefined) {
+        return false;
+      }
+
+      return itemId;
     },
-    onStart({ data }) {
-      setDragItemId(data);
+    onStart({ data: itemId }) {
+      const item = items.find((item) => item.id === itemId);
+      if (items.at(-1) === item && item?.source === "local") {
+        toast.error("Local token is always last and can not be moved");
```

Comment:
Local style, not token

Reply:
This is the wording that I chose. If you want we can change it to "Local style source" but not "style" by itself

Reply:
or perhaps more semantic: "Local styles are always last and can not be moved."

Reply:
works, thanks